engine/src/org/pentaho/di/trans/steps/exceloutput/ExcelOutput.java

```diff
@@ -681,7 +681,7 @@ private boolean closeFile() {
         }
 
       }
-      // data.formats.clear();
+      data.formats.clear();
```

Comment:
@mbatchelor I double-checked this step Excel Output. My fix does not affect options: append and append to Excel template. After merging PR I going to add this affected area in jira and QA will check it one more time.